config/packages/dev/jms_serializer.yaml

```diff
@@ -0,0 +1,7 @@
+jms_serializer:
```

Comment:
This config is missing the naming strategy. Right now the serializer by default will create output names (JSON) with underscores and lower case. But in the past we expected camelcase. (no cap start).

Add:
`property_naming:
        id: 'jms_serializer.identical_property_naming_strategy'`

Reply:
Yes, this comment should go in the non-dev config file.